solidity/contracts/AugurConstantProductMarketRouter.sol

```diff
@@ -453,6 +435,22 @@ contract AugurConstantProductRouter {
 		return marketList.length;
 	}
 
+	function getUserLpTokenIdsForMarket(address augurMarketAddress, address user) external view returns (uint256[] memory) {
```

Comment:
should this have paging? Is there a risk that an LP could have too many of these?